boards/arm/b_g474e_dpow1/b_g474e_dpow1.dts

```diff
@@ -7,6 +7,7 @@
 /dts-v1/;
 #include <st/g4/stm32g474Xe.dtsi>
 #include <st/g4/stm32g474r(b-c-e)tx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/adc/adc.h>
```

Comment:
Please move this to "dts/am/st/g0/stm32g0.dtsi"

Reply:
Done